$MirOS: ports/multimedia/livestream/patches/patch-liveMedia_FileSink_cpp,v 1.2 2005/05/19 20:06:17 tg Exp $
--- liveMedia/FileSink.cpp.orig	Fri Dec 23 13:00:41 2005
+++ liveMedia/FileSink.cpp	Wed Dec 28 18:02:48 2005
@@ -92,8 +92,8 @@ void FileSink::addData(unsigned char* da
 		       struct timeval presentationTime) {
   if (fPerFrameFileNameBuffer != NULL) {
     // Special case: Open a new file on-the-fly for this frame
-    sprintf(fPerFrameFileNameBuffer, "%s-%lu.%06lu", fPerFrameFileNamePrefix,
-	    presentationTime.tv_sec, presentationTime.tv_usec);
+    sprintf(fPerFrameFileNameBuffer, "%s-%llu.%06lu", fPerFrameFileNamePrefix,
+	    (long long)presentationTime.tv_sec, presentationTime.tv_usec);
     fOutFid = OpenOutputFile(envir(), fPerFrameFileNameBuffer);
   }
 
